README.md

```diff
@@ -21,27 +21,19 @@
 
 *Support for Table accounts in CosmosDB is coming soon!
 
-## Mongo "Scrapbooks"
+![Browse CosmosDB and MongoDB databases](resources/Browse.png)
 
-- Configure the user setting `mongo.shell.path` to your mongo shell executable path
-- Click on any DB to open the Mongo shell playground editor
-- Enter your scripts, eg: `db.<collectionName>.find()`
-- IntelliSense (completions) will be provided as you write your scripts
-- Select the script and press `CMD+"` (`CTRL+"` on Windows and Linux) to see the results
+## Mongo "Scrapbooks" (Run Mongo Commands with Rich Intellisense)
 
-## Managing Azure Subscriptions
-
-If you are not signed in to Azure, you will see a "Sign in to Azure..." link. Alternatively, you can select "View->Command Palette" in the VS Code menu, and search for "Azure: Sign In".
-
-![Sign in to Azure](resources/SignIn.gif)
-
-If you don't have an Azure Account, you can sign up for one today for free and receive $200 in credits by selecting "Create a Free Azure Account..." or selecting "View->Command Palette" and searching for "Azure: Create an Account".
+- Attach to a MongoDB database account

```

Comment:
How about "View your MongoDB database account by signing in to Azure or attaching with a connection string"

If you just say "Attach" that leaves out the Azure scenario